internal/engine/source/activitypub/task.go

```diff
@@ -0,0 +1,71 @@
+package activitypub
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/rss3-network/node/internal/engine"
+	"github.com/rss3-network/node/provider/activitypub"
+	activityx "github.com/rss3-network/protocol-go/schema/activity"
+	"github.com/rss3-network/protocol-go/schema/network"
+	"github.com/rss3-network/protocol-go/schema/typex"
+	"go.uber.org/zap"
+)
+
+var _ engine.Task = (*Task)(nil)
+
+// TODO: should be pulled from VSL (NetworkParams contract)
```

Comment:
Should implement the related logics here, we can set NetworkParams in testnet contract.

Reply:
And it should be moved to `option.go`?

Reply:
I moved the initialization of mastodon's `TimestampStart` value to option.go using the value retrieved from VSL. Also, I implemented the logic to properly use it here.